app/templates/components/session-card.hbs

```diff
@@ -0,0 +1,36 @@
+<div class="event wide ui grid row">
```

Comment:
@CosmicCoder96 `wide` class is required here. If we remove it the ui distorts. Please check the screenshot
![screenshot from 2017-06-27 02-55-11](https://user-images.githubusercontent.com/12807846/27561016-11cf73b0-5ae4-11e7-85f4-96bd2321d326.png)